coral/docs/development-with-remote-api.md

```diff
@@ -24,10 +23,10 @@
    4.1. If you have not setup a remote API mode, please follow [First setup](../docs/development-with-remote-api.
    md#first-setup)
    4.2. If you already have a setup, run `pnpm dev`
+5. [Create new vite mode for remote API development](#create-new-vite-mode-for-remote-api-development)
```

Comment:
Should vite always be in backticks like `vite` ?

Reply:
Good question. I'm afraid I'm a bit inconsistent with that. I think backticks should be used for e.g. filenames and directories. I'm unsure if they should be used in technical names like "vite" or "React" 🤔

Reply:
I think that we're referring to the name of the library here, so probably it should not be in backticks. However, I'm sure that it should always be capitalised as "Vite" ^^ Like it is in the docs https://vitejs.dev/guide/

Reply:
🫡 :D

Reply:
Updated all to Vite with capital V.